src/CLI/klish/patches/klish-2.1.4/clish/shell/shell_tinyrl.c.diff

```diff
@@ -95,35 +74,35 @@
 >     //reset_check_type(false);
 > 
 >  	/* keep compiler happy */
-225a288,292
+225a267,271
 > 	char custom_errmsg[50];
 > 	clish_ptype_method_e method;
 > 	const clish_ptype_t *ptype = NULL;
 > 	const clish_param_t *failed_param = NULL;
 > 	int cnt = 0;
-230a298,300
+230a277,279
 > 	/* Renew prompt */
 > 	clish_shell_renew_prompt(shell);
 > 
-237a308,313
+237a287,292
 > 	clish_shell_tinyrl_complete_ignore_error(this);
 > 	line = tinyrl__get_line(this);
 > 	//if(try_alias_execution(shell, line)) {
```

Comment:
Still adding a bunch of commented-out lines throughout these patch files. Please just remove them to try to enhance readability (this diff format is hard to read enough! :) )